runtime/src/accounts.rs

```diff
@@ -1325,9 +1325,19 @@ mod tests {
         assert_eq!(loaded_accounts[0], Err(BankError::AccountLoadedTwice));
     }
 
+    fn get_tmp_accounts_path(paths: &str) -> String {
```

Comment:
This could be a macro with file! and line! to eliminate that paths parameter and the flaky tests that'd result from copy pasta.

Reply:
agreed, can copy pasta this from blocktree ;)